BuildingSync.xsd

```diff
@@ -3557,24 +3550,24 @@
                   <xs:element name="Replacement" minOccurs="0">
                     <xs:complexType>
                       <xs:sequence>
-                        <xs:element name="ExistingSystemReplaced" minOccurs="0" maxOccurs="unbounded">
+                        <xs:element name="ExistingSystemIDReplaced" minOccurs="0" maxOccurs="unbounded">
                           <xs:annotation>
-                            <xs:documentation>ID numbers of any existing systems replaced by the measure.</xs:documentation>
+                            <xs:documentation>ID numbers of any existing systems replaced by the measure. Valid reference paths: BuildingSync/Facilities/Facility/Systems/***Systems/***System.</xs:documentation>
```

Comment:
what are the `***Systems` denoting?

Reply:
Any valid system according to the path. Is there a better way to interpret this?

Reply:
shouldn't it be `BuildingSync/Facilities/Facility/***Systems/***System` then? I think there are redundant `/Systems`

Reply:
There is a `auc:Systems` element before all child systems, isn't it?

Reply:
Oh I see, the `***Systems` is `LightingSystems`, `DishwasherSystems`, etc. There are a few that are not ***System but we can ignore that for now (e.g., Pools, PlugLoads, etc).

Leave it as is, thanks!